### DISCUSSION TITLE
Compiled with problems

### DISCUSSION THREAD

**Discussion by Alexva397** [weareseeed/react-square-web-payments-sdk#30] [Help] [answered]
Hello, I am trying to implement basic usage of the SDK in a react app. I used [THIS](https://www.youtube.com/watch?v=GNo9y1CRcZ4) that square published to youtube as a guide. when I install my dependencies 
`yarn add react-square-web-payments-sdk @square/web-sdk`
and attempt to use the library like so: 
```import { SquarePaymentsForm, CreditCardInput } from 'react-square-web-payments-sdk';

function App() {
  return (
    <div className="App">
      <SquarePaymentsForm
        applicationId='xxxx'
        locationId='xxxx'
        cardTokenizeResponseReceived={(token) => {
          console.log(token);
        }}
      >
        <CreditCardInput/>
      </SquarePaymentsForm>
    </div>
  );
}

export default App;
```

I receive the following errors:

![Screen Shot 2022-01-21 at 07 44 39 (2)](https://user-images.githubusercontent.com/72521427/150538884-c2ba1d96-4d19-4b5f-8e87-54f4b024ff6c.png)

Any help would be greatly appreciated!

Thanks
Alex V

**Comment by danestves** [comment, marked as answer]
Hi @Alexva397 can you make a reproduction of the error here 👉 https://next.new/, so we can give you more support about it reading the code

**Reply by Alexva397** [reply]
Thanks for the response Dan. I tried recreating the issue in a stackblitz next.js app and on codesandbox in a react app, with no luck. Must be something I am doing locally when attempting to use the package. The error only presents itself when attempting to import:
`import { SquarePaymentsForm } from 'react-square-web-payments-sdk';`

**Reply by danestves** [reply]
I'm making a reproduction from my side, are you using `next-transpile-modules`?

**Reply by Alexva397** [reply]
Not in the react app, which is what my final implementation will be in. Is this required?

**Reply by danestves** [reply]
If you are using `next` is required to use it, but I see that you are using a plain `create-react-app` that I think is not supporting `esm` modules, can you run `npm install react-square-web-payments-sdk@canary` and try with that version?

**Reply by Alexva397** [reply]
Sorry for the late response, Unfortunately the same issue, with a slightly different error message.

![Screen Shot 2022-01-27 at 18 29 31 (2)](https://user-images.githubusercontent.com/72521427/151466152-584353a8-87cd-4905-ad2e-540f681481f8.png)

**Reply by johnbona** [reply]
@Alexva397 I too encountered the same issue (it's Webpack 5) and created a PR for it: https://github.com/square/web-sdk/pull/67

**Reply by Alexva397** [reply]
Thanks for the help! @johnbona @danestves
